external_proto/README.md

```diff
@@ -0,0 +1,10 @@
+These are protos copied unchanged from https://github.com/googleapis/googleapis
```

Comment:
`third_party` can have Google code. See the tf example I mentioned. The lines of what is owned by Google and isn't owned can get blurry and change.

I think it would be simpler to treat all of these protos the same, in the same directory.

Reply:
> I think it would be simpler to treat all of these protos the same, in the same directory.

Not sure what you're proposing here, but I'll rename `external_proto` to `third_party/googleapis` for now. (I definitely don't want the google.type etc protos to end up in the same library as the event protos, or that should at least be easy to avoid. They already exist in packages we can depend on, and generating the same type in multiple packages can cause issues.)